src/here-studio.ts

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env node
+
+/*
+  Copyright (C) 2018 - 2019 HERE Europe B.V.
+  SPDX-License-Identifier: MIT
+
+  Permission is hereby granted, free of charge, to any person obtaining
+  a copy of this software and associated documentation files (the
+  'Software'), to deal in the Software without restriction, including
+  without limitation the rights to use, copy, modify, merge, publish,
+  distribute, sublicense, and/or sell copies of the Software, and to
+  permit persons to whom the Software is furnished to do so, subject to
+  the following conditions:
+
+  The above copyright notice and this permission notice shall be
+  included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
+  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+  IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+  CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+  TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+import * as common from './common';
+import {ApiError} from "./api-error";
+import * as _ from "lodash";
+
+import * as fs from "fs";
+
+import * as program from 'commander';
+
+import {getSpaceDataFromXyz, uploadToXyzSpace, handleError, execute, createSpace} from "./xyzCommon";
+
+const prompter = require('prompt');
+const commands = ["list", "clone", "open", "show", "delete"];
+
+program
+    .version('0.1.0');
+
+
+program
+    .command("list")
+    .description("information about available xyz studio projects")
+    .action(async function (options) {
+        listProjects(options)
+            .catch((error: any) => {
+                handleError(error);
+            })
+    });
+
+program
+    .command("delete <project-id>")
+    .description("delete the project with the given id")
+    .action(async (geospaceId, options) => {
+        deleteProject(geospaceId, options)
+            .catch((error) => {
+                handleError(error, true);
+            })
+    });
+
+program
+    .command("show <project-id>")
+    .description("open the project with the given id")
+    .action(async (geospaceId, options) => {
+        showProject (geospaceId, options)
+            .catch((error) => {
+                handleError(error);
+            })
+    });
+
+program
+    .command("clone <project-id>")
+    .description("open the project with the given id")
```

Comment:
Description will be changed to "clone the project with the given id or viewerUrl"